tools/ports/freetype.py

```diff
@@ -7,21 +7,19 @@
 import shutil
 from pathlib import Path
 
-TAG = 'version_1'
-HASH = '0d0b1280ba0501ad0a23cf1daa1f86821c722218b59432734d3087a89acd22aabd5c3e5e1269700dcd41e87073046e906060f167c032eb91a3ac8c5808a02783'
-
```

Comment:
flake8 doesn't like that you removed this line I'm afraid.

Reply:
It looks like we have several other failures, so I'll bundle this with another fix (if needed).

Looking at the failures, it's not immediately clear if this is a problem with the tests or freetype.
test-core0, test-core2, and test-core3 all look like test failures.
test-mac and test-other could be test errors. In the changelog, it looks like 2.7 changed subpixel hinting, and 2.8 fixed some bugs with that.
https://freetype.org/index.html#news